src/libraries/SettingsLib.sol

```diff
@@ -34,4 +34,11 @@ library SettingsLib {
             _hook := and(settings, mask)
         }
     }
+
+    function isExpired(Settings settings) internal view returns (bool _isExpired) {
+        uint256 _exp = expiration(settings);
+        // Keys with expiry of 0 never expire
+        if (_exp == 0) return false;
+        return _exp < block.timestamp;
```

Comment:
so its not inclusive? if you set a key to expire at 10, and it IS 10 does that mean it should be false?

Reply:
just updated to be inclusive